src/libraries/pkg/test/testPackages.proj

```diff
@@ -13,9 +13,13 @@
     <!-- support override via commandline -->
     <RuntimesToInclude Condition="'$(RuntimesToInclude)' != ''" Include="$(RuntimesToInclude)" />
     <TargetFrameworksToInclude Condition="'$(TargetFrameworksToInclude)' != ''" Include="$(TargetFrameworksToInclude)" />
-
+    <SymbolsPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.symbols.nupkg" />
+    <ExcludePackagesList Include="@(SymbolsPackages)" />
+    <ExcludePackagesList Include="@(ExcludePackages->'$(LibrariesAllConfigShippingPackagesDir)%(Identity).$(Version).nupkg')" />
+    <TestPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.nupkg" Exclude="@(ExcludePackagesList)" Condition="'$(PackagesToTest)' == ''"/>
```

Comment:
Do we run package tests on non shipping packages?

Reply:
We currently don't run package testing for transport packages which is currently true for all the packages.